shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1121,23 +1138,98 @@ private boolean performCursorMoveAction(
           return true;
         }
         break;
+      case AccessibilityNodeInfo.MOVEMENT_GRANULARITY_LINE:
+      case AccessibilityNodeInfo.MOVEMENT_GRANULARITY_PARAGRAPH:
+      case AccessibilityNodeInfo.MOVEMENT_GRANULARITY_PAGE:
+        return true;
```

Comment:
We don't have semantics action for these actions. I feel weird to add them, too, because they aren't really going to be used. voice access only care about the action return true and the semantics node gets updated immediately, which is done by the predictCursorMovement.

If we do want to add them, I feel we should just merge all the cursor movement action into one semantics action that takes in granularity and direction.